### DISCUSSION TITLE
Lines plugin broken again?

### DISCUSSION THREAD

**Discussion by ezeholz** [lowlighter/metrics#494] [Q&A] [answered]
As we talked in https://github.com/lowlighter/metrics/discussions/457, Lines plugin got fixed in 3.13
But 2 days ago, my stats went from `25.6k added, 9.17k removed` to `0 added, 0 removed`
Seems to be a general problem, because the plugin seems to be struggling too in the metrics readme.
![image](https://user-images.githubusercontent.com/30635862/130383134-89e7e61f-af83-47b8-8a6b-b110867a3ef2.png)
Could be something with GitHub API?

**Comment by Nixinova** [comment]
Doesn't seem to be metrics' fault, it [broke yesterday](https://github.com/Nixinova/Nixinova/commit/55c5636568c8ef7a9c04e1d887a60e0c4bef5c30#diff-15c77d2bbedf6bd69a53f23e8705a32acfeeb9a45ca324f78e8f385924ab3f29R1399) and the last update to this repo was 3d ago.

**Comment by ezeholz** [comment, marked as answer]
Seems to be a problem with GitHub API, now it's working as expected on my end.
